### DISCUSSION TITLE
paragraphs

### DISCUSSION THREAD

**Discussion by mariannevanharten** [FamilySearch/GEDCOM#531] [Q&A] [answered]
My family tree application (webtrees) currently stores the data in Gedcom 5.5.1
For now I write my data in 'dual Gedcom' (5.5.1 and also 7.0)
I want to write a song into my notes. The song consist of some short lines
How do I avoid the lines of the song concatenating?

Best regards,
Marianne

**Comment by Norwegian-Sardines** [comment, marked as answer]
Marianne,
I’m not an expert on v7 GEDCOM layout, but maybe this image will help!
![IMG_0720](https://github.com/user-attachments/assets/d95153fb-e912-4441-ab99-636fc8307dd0)

My understanding is that the `NOTE.CONC` has been deprecated in v7 but the `NOTE.CONT` still remains.  This works like a new paragraph.

I'm not sure you can have a `NOTE` work in both v5.5.1 and v7 because of this deprecation, but webtrees may work because Greg knows about this and I think he stores his `NOTE` data with the lines concatenated!
